core/deployment/src/main/java/io/quarkus/deployment/steps/ConfigurationSetup.java

```diff
@@ -259,11 +264,7 @@ void finalizeConfigLoader(
             Consumer<BytecodeRecorderObjectLoaderBuildItem> objectLoaderConsumer,
             List<ConfigurationCustomConverterBuildItem> converters,
             List<RunTimeConfigurationSourceBuildItem> runTimeSources) {
-        final ClassOutput classOutput = new ClassOutput() {
-            public void write(final String name, final byte[] data) {
-                classConsumer.accept(new GeneratedClassBuildItem(false, name, data));
-            }
-        };
+        final ClassOutput classOutput = (name, data) -> classConsumer.accept(new GeneratedClassBuildItem(false, name, data));
```

Comment:
So I would have merged this one already... if the "let's add lambdas everywhere" part was out of it.

We are very cautious about where we add lambdas and it requires thinking twice about it.

Could you make your PR only about the proposed change?

Reply:
Sorry, I pre-configure some Auto Save Actions, which caused this. Have just disabled, for future.